src/components/lyne-toast/animations/toast.enter.ts

```diff
@@ -0,0 +1,29 @@
+import { InterfaceAnimation } from '../../../global/core/components/animations/animation-interface';
+import { createAnimation } from '../../../global/core/components/animations/animation';
+
+export const toastEnterAnimation = (baseEl: HTMLElement, position: string): InterfaceAnimation => {
+  const baseAnimation = createAnimation();
+  const wrapperAnimation = createAnimation();
+
+  const root = baseEl.shadowRoot;
+  const wrapperEl = root.querySelector('.toast-wrapper') as HTMLElement;
+
+  wrapperAnimation.addElement(wrapperEl);
+
+  switch (position) {
+    case 'top':
+      wrapperAnimation.fromTo('opacity', 0.01, 1);
+      wrapperAnimation.duration(750);
```

Comment:
durations should be taken from design tokens, but probably such a long duration does not exist right now, if it really should be that slow, we can ask @mcilurzo and add some more tokens in lyne-design-tokens

Reply:
Resolved using variables '--animation-easing', '--animation-duration-6x' and '--animation-duration-3x'.

Added a FIXME:
@mcilurzo could be possible remove the 'ms' suffix for '--animation-duration-n' values?

Reply:
Opened a discussion in lyne teams chat.

Instead of parsing the css variables here it's also possible to directly import the tokens in javascript (but still a string with unit): 

`import {
  AnimationDuration6x
} from 'lyne-design-tokens/dist/js/tokens.es6';

Reply:
Importing javascript would be better in terms of performance for a consumer, but less flexible when a variable should change at runtime. This would probably not be a problem here as we don't expect overridden animation times during runtime(?). I tend to use the solution you chose because we are sure we can react to runtime changes

Reply:
I like it, thank you :)